phpBB/phpbb/mention/source/group.php

```diff
@@ -15,7 +15,7 @@
 
 class group extends base_group
 {
-	/** @var string|false */
+	/** @var int */
```

Comment:
in base_group cache_ttl is string|false. I dont remember if type override is possible in php